sphinx-docs/Lateral-Movement-Guide.md

```diff
@@ -0,0 +1,92 @@
+Windows Lateral Movement Guide
+======================
+
+Using CALDERA's lateral movement and execution abilities, you can perform and test an adversary's capability to move 
```

Comment:
language suggestion for the opening -- something like the following might be a bit more concise: 

"Exercising Caldera's lateral movement and remote execution abilities allows you to test how easily an adversary can move within your network."